packages/client/utils/SlackManager.ts

```diff
@@ -213,9 +224,9 @@ abstract class SlackManager {
     return this.post<PostMessageResponse>('https://slack.com/api/chat.postMessage', payload)
   }
 
-  updateMessage(channelId: string, text: string, ts: string) {
+  updateMessage(channelId: string, blocks: string | Array<{type: string}>, ts: string) {
```

Comment:
blocks must be a string (you pass it in correctly, but the type can't be an array)

Reply:
Thanks! I've moved the stringify to `updateMessage` so that we can have the same call signature here and keep the array type